src/SDKs/DataMigration/Management.DataMigration/Generated/DataMigrationServiceClient.cs

```diff
@@ -46,30 +46,26 @@ public partial class DataMigrationServiceClient : ServiceClient<DataMigrationSer
         /// </summary>
         public ServiceClientCredentials Credentials { get; private set; }
 
-        /// <summary>
-        /// Version of the API
-        /// </summary>
-        public string ApiVersion { get; private set; }
```

Comment:
Why is this being deleted?

Reply:
I am generating using the latest Autorest (2.0.4284) from the public swagger where the Client does contain ApiVersion.  

autorest "https://github.com/Azure/azure-rest-api-specs/blob/master/specification/datamigration/resource-manager/readme.md" --csharp --csharp-sdks-folder=F:\Git\Sync\azure-sdk-for-net\src\SDKs --version="latest" --reflect-api-versions

Is there a specific autorest version which I should be using?

Reply:
Confirmed this change is expected